Gemfile.lock

```diff
@@ -417,20 +417,20 @@ GEM
       coercible (~> 1.0)
       descendants_tracker (~> 0.0, >= 0.0.3)
       equalizer (~> 0.0, >= 0.0.9)
-    warden (1.2.8)
-      rack (>= 2.0.6)
+    warden (1.2.9)
+      rack (>= 2.0.9)
     webmock (2.3.2)
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (4.2.2)
+    webpacker (4.3.0)
       activesupport (>= 4.2)
       rack-proxy (>= 0.6.1)
       railties (>= 4.2)
-    websocket-driver (0.7.2)
+    websocket-driver (0.7.3)
```

Comment:
### [websocket-driver](https://github.com/faye/websocket-driver-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.7.2 → 0.7.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/faye/websocket-driver-ruby/compare/0.7.2...0.7.3).

These are the individual commits:
- (e87bfef) [Accept HTTP responses that have an empty reason-phrase, per …](https://github.com/faye/websocket-driver-ruby/commit/e87bfef9bc0013e32546f2b8d158d20212915076)
- (eb643f5) [Bump version to 0.7.3](https://github.com/faye/websocket-driver-ruby/commit/eb643f59499dec53939b30eebba5ca2f5e7296a7)


</details>